language/move-lang/README.md

```diff
@@ -0,0 +1,136 @@
+---
+id: move-lang
+title: Move Source Language
+custom_edit_url: https://github.com/libra/libra/edit/master/language/move-lang/README.md
+---
+
+## Code under this subtree is experimental. It is out of scope for the Libra Bug Bounty until it is no longer marked experimental.
+
+# Move Source Language
+
+## Summary
+
+Move source language is an ergonomic language for writing Modules and Scripts that compile to Move bytecode.
+
+## Overview
+
+The purpose of the Move source language is to make writing Move Modules and Scripts easier than through the Mover IR or Move bytecode.
```

Comment:
How about the following rephrasing of the overview?

Move source language is an expression-based language that aims to simplify writing Move programs ---modules and scripts--- without hiding the core concepts in Move bytecode.